packages/core/src/App/Containers/RealAccountSignup/real-account-signup.jsx

```diff
@@ -274,10 +261,13 @@ class RealAccountSignup extends Component {
 export default connect(({ ui, client }) => ({
     available_crypto_currencies: client.available_crypto_currencies,
     can_change_fiat_currency: client.can_change_fiat_currency,
+    can_upgrade_to: client.can_upgrade_to,
     has_real_account: client.has_active_real_account,
     currency: client.currency,
     is_real_acc_signup_on: ui.is_real_acc_signup_on,
     closeRealAccountSignup: ui.closeRealAccountSignup,
     setParams: ui.setRealAccountSignupParams,
+    residence: client.residence,
+    is_im_residence: client.residence === 'im',
```

Comment:
is it neater to define this as a computed value?

Reply:
I have left this here because I did not wanted client store to be crowded with small computed values, since we don't know yet which other values are going to be needed. My suggestion is to keep it this way and and the end of the round trip for EU support, we can then refactor. I'll add a todo in here, if you agree.

Reply:
Sure! keep going